KeyAuth.md

```diff
@@ -0,0 +1,15 @@
+# HTTPS with client certificate authentication
+
+To protect your openHAB instance in the public Internet network we can use client authenticate with SSL certificate feature.
+
+In order to prepare environment to work with this feature we must to do some below steps:
+
+## Own Certificate Authority
+TODO
```

Comment:
Yes, I have such plans. I will try do it in next weekend.

I think that the best solutions is put links to forum and to documentation page, like:
https://community.openhab.org/t/using-nginx-reverse-proxy-authentication-and-https/14542 
http://docs.openhab.org/configuration/nginx.html

And next add information how to prepare and configure client certificate into those page.